store/tikv/coprocessor.go

```diff
@@ -977,6 +1001,28 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *RPCCon
 			}
 		}
 	}
+	if resp.pbResp.IsCacheHit && cacheValue != nil {
```

Comment:
Is it possible that cache is hit but cache value is nil?

Reply:
If it's not possible, we better not check it, and let it panic.

Reply:
It is not possible, only to protect from buggy TiKV implementation

Reply:
The code was written considering that the cache layer should never break normal requests. How about adding warnings for it?

Reply:
If the cache hit, the response data would be empty, how to handle it?

Reply:
You are right. It can not be handled as well. Shall we write something like:

```go
if cacheValue == nil {
    return nil, errors.New("Internal error: received illegal TiKV response")
}
```